sources/packages/backend/libs/integrations/src/services/sfas/sims-to-sfas.service.ts

```diff
@@ -119,6 +127,128 @@ export class SIMSToSFASService {
     return modifiedStudentIds;
   }
 
+  /**
+   * Get all student ids of students who have one or more updates in application related data.
+   * @param modifiedSince the date after which the student data was updated.
+   * @param modifiedUntil the date until which the student data was updated.
+   */
+  async getAllStudentsWithApplicationUpdates(
+    modifiedSince: Date,
+    modifiedUntil: Date,
+  ): Promise<ApplicationRecord[]> {
+    return (
+      this.applicationRepo
+        .createQueryBuilder("application")
+        .select("student.id", "studentId")
```

Comment:
minor. please select the application id first as we are going from application here.